cmd/thanos/query.go

```diff
@@ -179,6 +180,8 @@ func registerQuery(app *extkingpin.App) {
 		Default(string(dns.MiekgdnsResolverType)).Hidden().String()
 
 	unhealthyStoreTimeout := extkingpin.ModelDuration(cmd.Flag("store.unhealthy-timeout", "Timeout before an unhealthy store is cleaned from the store UI page.").Default("5m"))
+	ignoreStoreErrors := cmd.Flag("store.ignore-errors", "Ignore errors from store endpoints. This is for qeury result completeness.").
```

Comment:
what's the reason to ignore errors from store? not clear on the rational from the comment

Reply:
here store doesn't restrict to pantheon-store, but any components enable storeAPI, if this is a feature flag control, then ignore my comment

Reply:
Correct. I found in Thanos code base that `store` refers to both Store and DB which both provide time series streaming API.
Yes, it's a feature flag. I'll update the flag summary.